src/Athena/Views/Shared/_Layout.cshtml

```diff
@@ -2,70 +2,111 @@
-                    <span class="icon-bar"></span>
-                    <span class="icon-bar"></span>
-                    <span class="icon-bar"></span>
-                </button>
-                <a asp-area="" asp-controller="Home" asp-action="Index" class="navbar-brand">Athena</a>
-            </div>
-            <div class="navbar-collapse collapse">
-                <ul class="nav navbar-nav">
-                    <li><a asp-area="" asp-controller="Home" asp-action="Index">Home</a></li>
-                    <li><a asp-area="" asp-controller="Home" asp-action="About">About</a></li>
-                    <li><a asp-area="" asp-controller="Home" asp-action="Contact">Contact</a></li>
+    <div class="navbar-fixed indigo darken-4">
+        <nav>
+            <div class="nav-wrapper indigo darken-4">
+                <img src="~/images/owl icon.svg" class="left responsive-img" style="max-height:96px; max-width:96px; padding-left: 3em" />
+
+                <a href="#" data-activates="mobile-demo" class="right button-collapse"><i class="material-icons">menu</i></a>
+                <ul class="right hide-on-med-and-down">
+                    <li><a href="NewAccountPage">New Account<i class="material-icons right">person_add</i></a></li>
+                    <li><a href="Login_Page">Login<i class="material-icons right">person_outline</i></a></li>
+                </ul>
+                <ul class="side-nav" id="mobile-demo">
+                    <li><a href="NewAccountPage">New Account</a></li>
+                    <li><a href="Login_Page">Login</a></li>
                 </ul>
+                <h2 class="brand-logo no-pad-top" style="margin-top:12px;font-size:3em;font-weight:bold;font-family:'Palatino Linotype'">Athena</h2>
             </div>
-        </div>
-    </nav>
-    <div class="container body-content">
-        @RenderBody()
-        <hr />
-        <footer>
-            <p>&copy; 2018 - Athena</p>
-        </footer>
+        </nav>
     </div>
+        <div class="container body-content;">
+            @RenderBody()
+            <hr />
+            <footer>
+                <p>&copy; 2018 - Athena</p>
+            </footer>
+        </div>
 
-    <environment include="Development">
-        <script src="~/lib/jquery/dist/jquery.js"></script>
-        <script src="~/lib/bootstrap/dist/js/bootstrap.js"></script>
-        <script src="~/js/site.js" asp-append-version="true"></script>
-    </environment>
-    <environment exclude="Development">
-        <script src="https://ajax.aspnetcdn.com/ajax/jquery/jquery-2.2.0.min.js"
-                asp-fallback-src="~/lib/jquery/dist/jquery.min.js"
-                asp-fallback-test="window.jQuery"
-                crossorigin="anonymous"
-                integrity="sha384-K+ctZQ+LL8q6tP7I94W+qzQsfRV2a+AfHIi9k8z8l9ggpc8X+Ytst4yBo/hH+8Fk">
+        <script type="text/javascript" src="~/athena.js"></script>
+
+        @RenderSection("Scripts", required: false)
+
+        <script>
```

Comment:
Pull these scripts to their own files to be included by the bundler. The only reason we should have inline scripts in views is if it needs dynamic data from the model.